### DISCUSSION TITLE
orange pi doesnt turn off monitor?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#76] [General]
its not ubuntu 22.04 problem, more of a general thing with this sbc bcos its the same with ambian,debian, ubuntu... but i got most help around here so ill ask.
when you set display time out and it goes black, it does exactly that, goes black but my ips monitor continues glowing(backlight on) bcos well ips, its not off like it should be. 
can this be fixed somehow? bcos of that i set display timeout to never and turn off the monitor by button when not using it.

**Comment by Joshua-Riek** [comment]
That might be something with your monitors, I have a few ips displays, and they all turn their backlight off with the display times out.

However, I know there is an issue where a brightness slider shows up in the user interface, this is because there are two pwm-backlight devices defined in the device tree by default. Now, this sort of backlight is meant for an LCD screen connected to the opi5 with a ribbon cable, not an HDMI monitor. I don't know if this has any correlation, but software-wise, its the first thing I can think of.

**Comment by Gruttebier** [comment]
First a big thanks for all this work. 

I have the same, the screen only turns black. With the Orange pi debian os it did work. My screen is an iiyama. 

If i could i would buy you a cup of coffee.
